miqa/core/tasks.py

```diff
@@ -25,7 +26,14 @@
 from miqa.learning.nn_classifier import evaluate1
 
 
-def evaluate_data(images: List[Image], session: Session):
+@app.task
```

Comment:
There's a preferred shortcut for this:

```python
from celery import shared_task
```

Then you can just decorate with `@shared_task`, and they'll be automatically picked up by the app. https://docs.celeryproject.org/en/stable/django/first-steps-with-django.html#using-the-shared-task-decorator